tests/system/runtimes/test_nuclio.py

```diff
@@ -47,3 +51,48 @@ def test_deploy_function_with_error_handler(self):
 
         self._logger.debug("Deploying nuclio function")
         function.deploy()
+
+    def test_serving_with_child_function(self):
```

Comment:
You're using v3io in this test, meaning it can only run in the enterprise system tests CI
so you need to add here `` (see https://github.com/mlrun/mlrun/blob/development/CONTRIBUTING.md#adding-system-tests)
Note that the other tests in this class are compatible with open source, so as mentioned in the link you'll have to specify the skip on every test method (instead of one time on the class)